docs/policy.json.md

```diff
@@ -0,0 +1,251 @@
+<!-- NOTE: Keep this in sync with signature/policy_types.go! -->
```

Comment:
We want to support md2man format. HTML comment syntax breaks conversion. Need to add man headers per #121. For example:

```
% POLICY.JSON(5) policy.json Man Page
% Miloslav Trmac
% September 2016
```